pom.xml

```diff
@@ -245,6 +247,11 @@
                 <version>${version.mysql.driver}</version>
             </dependency>
             <dependency>
```

Comment:
Will this also be added to the plug-in ZIP?

Reply:
Yes, it will be added too - as it is a non-`provided` dependency.